src/main/java/wolox/reactortraining/controllers/TopicController.java

```diff
@@ -0,0 +1,56 @@
+package wolox.reactortraining.controllers;
+
+import static reactor.core.publisher.Mono.error;
+
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import wolox.reactortraining.dtos.TopicDto;
+import wolox.reactortraining.exceptions.TopicNotFound;
+import wolox.reactortraining.models.Topic;
+import wolox.reactortraining.repositories.TopicRepository;
+
+@RestController
+@RequestMapping("/topics")
+public class TopicController {
+
+    @Autowired
+    private TopicRepository topicRepository;
+
+    @PostMapping
+    public Mono<Void> create(@Valid @RequestBody TopicDto topicDto) {
+        return Mono
```

Comment:
could be simplified, the dto has the same fields as the entity, could directly make the insertion, and the repository returns the Mono<entity> as an answer.